content/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -176,6 +176,7 @@ Allows members to create and administrate all workspaces within the organization
 
 - Admin access to all workspaces. Any action that requires admin access is also available to teams with permission to manage workspaces.
 - The ability to create new workspaces (otherwise only available to organization owners).
+- Allows members to create, update and delete [Variable Sets](/cloud-docs/workspaces/variables/managing-variables#variable-sets)
```

Comment:
What do you think of this, just because the opening sentence says "allows members" already?

```suggestion
- Create, update, and delete [Variable Sets](/cloud-docs/workspaces/variables/managing-variables#variable-sets)
```